app/data/pencil2d.appdata.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>pencil2d</id>
+  <launchable type="desktop-id">pencil2d.desktop</launchable>
+  <name>Pencil2D</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <summary>2D animation</summary>
+  <description>
+    <p>
+      An easy, intuitive tool to make 2D hand-drawn animations.
+    </p>
+  </description>
```

Comment:
According to the documentation, the description tag should contain “a long description of this component”. Might be a good idea to elaborate a bit here. @pencil2d/developers @Jose-Moreno do we have anything we could reuse here atm? I thought I had seen something like that somewhere, but I can’t seem to find it. And I don’t think what’s on our website atm would work well for this, that’s more like some sort of catchphrases in terms of how it’s worded.

Reply:
The GSOC description might work here. I'll wait and see what @Jose-Moreno thinks though.

Reply:
@scribblemaniac @J5lx Apologies This never reached me.
This is what was used for GSOC. 
> Pencil2D is a 2D animation, drawing & painting program that lets you easily create traditional handdrawn animation using both bitmap and vector graphics.

I'd probably shorten it to this for the metadata:
> Pencil2D is a 2D animation program that lets you easily create hand-drawn graphics using both bitmap and vector graphics.